packages/app/src/runner/event-manager.ts

```diff
@@ -150,7 +147,7 @@ export class EventManager {
     })
 
     this.ws.on('watched:file:changed', () => {
-      this.studioRecorder.cancel()
+      useStudioStore().cancel()
```

Comment:
I'm not very familiar with `pinia` so maybe this is not an issue, but I notice other `use*Store` usages in this file set the store into a shared const rather than making repeated `use*Store` calls. Just a style preference, or is there a functional difference?

Reply:
I just did a full text search + replace - `s/this.studioRecorder/useStudioStore()/g`. Was trying to minimise code changes, but since this seems like a fairly straight forward and obvious improvement, I'll go ahead and make it 👍

Reply:
Done. Thanks :+1: